otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template_supcon.yaml

```diff
@@ -0,0 +1,64 @@
+# Description.
```

Comment:
This is same with template.yaml. OTX TF have agreed that one template.yaml per one model. You can generate templates for your experiments, but that templates can be pushed.